perses/tests/test_samplers.py

```diff
@@ -109,7 +109,39 @@ def test_samplers():
             f.description = "Testing MultiTargetDesign sampler with %s transfer free energy from vacuum -> %s" % (testsystem_name, environment)
             yield f
 
+def test_hybrid_scheme():
+    """
+    Test ncmc hybrid switching
+    """
+    testsystem_names = ['AlanineDipeptideTestSystem']
+    niterations = 50 # number of iterations to run
+
+    # If TESTSYSTEMS environment variable is specified, test those systems.
+    if 'TESTSYSTEMS' in os.environ:
+        testsystem_names = os.environ['TESTSYSTEMS'].split(' ')
+
+    for testsystem_name in testsystem_names:
+        import perses.tests.testsystems
+        testsystem_class = getattr(perses.tests.testsystems, testsystem_name)
+        # Instantiate test system.
+        testsystem = testsystem_class()
+        # Test MCMCSampler samplers.
+        testsystem.environments = ['vacuum']
```

Comment:
Alternatively, we want to change this example to call the environment `vacuum` instead of `vacuum-imidazole`:
https://github.com/choderalab/perses/blob/master/perses/tests/testsystems.py#L1426-L1442